content/sensu-go/5.14/guides/securing-sensu.md

```diff
@@ -0,0 +1,265 @@
+echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","server auth","client auth"]}}}' > ca-config.json
+{{< /highlight >}}
+
+Then, using that CA, we can generate certificates and keys for each peer (backend server) by
+specifying their **Common Name (CN)** and their **hosts**. A `*.pem`, `*.csr` and `*.pem` will
+be created for each backend.
+
+{{< highlight shell >}}
+export ADDRESS=10.0.0.1,backend-1
+export NAME=backend-1
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
+
+export ADDRESS=10.0.0.2,backend-2
+export NAME=backend-2
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
+
+export ADDRESS=10.0.0.3,backend-3
+export NAME=backend-3
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" -profile=peer - | cfssljson -bare $NAME
+{{< /highlight >}}
+
+We will also create generate a *client* certificate that can be used by clients to connect to
+the etcd client URL. This time, we don't need to specify an address but simply a **Common Name
+(CN)** (here `client`). The files `client-key.pem`, `client.csr` and `client.pem` will be created.
+
+{{< highlight shell >}}
+export NAME=client
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=client - | cfssljson -bare $NAME
+{{< /highlight >}}
+
+If you have an enterprise license, then you can also generate a certificate for agent TLS authentication.
+
+{{< highlight shell >}}
+export NAME=agent-1
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=client - | cfssljson -bare $NAME
+{{< /highlight >}}
+
+See [etcd's guide to generating self signed certificates][11] for detailed instructions on securing etcd.
+
+Once done, you should have the following files created. The `*.csr` files will not be used in this guide.
+{{< highlight shell >}}
+agent-1-key.pem
+agent-1.csr
+agent-1.pem
+backend-1-key.pem
+backend-1.csr
+backend-1.pem
+backend-2-key.pem
+backend-2.csr
+backend-2.pem
+backend-3-key.pem
+backend-3.csr
+backend-3.pem
+ca-config.json
+ca-key.pem
+ca.csr
+ca.pem
+client-key.pem
+client.csr
+client.pem
+{{< /highlight >}}
+
+Hopefully you've found this useful! If you find any issues or have any questions, feel free to reach out in our [Community Slack][3], or [open an issue][4] on Github.
+
+<!-- LINKS -->
+[1]: /sensu-core/latest/guides/securing-sensu/
```

Comment:
Should this link to Sensu Go instead?
/sensu-go/latest/guides/securing-sensu/

Reply:
Double-check me, but it looks like this link is unused in the document body.